server/lib/utils/parse-connection.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Couldn't think of a better name than parse-connection, suggestions are welcome :). was thinking `extract-connection`, `get-client-info`... etc